tasks/metrics/mutant.rake

```diff
@@ -19,6 +19,13 @@ namespace :metrics do
       %W[--ignore #{matcher}]
     end
 
+    jobs =
+      if ENV.fetch('DEVTOOLS_SELF', false).eql?('1') && ENV.key?('CIRCLECI')
```

Comment:
> I think the whole `DEVTOOLS_SELF` can be removed.

I figured that handling for certain CIs for other users builds is a feature decision that would probably benefit from more discussion.